zeekpkg/manager.py

```diff
@@ -1477,6 +1695,14 @@ def __str__(self):
 
                         if dependency_node.info.package.matches_path(dep_name):
                             dependency_node.dependers[name] = dep_version
+                            _dep_name = dependency_node.info.package.git_url
+                            __dep_name = name_from_path(_dep_name)
+                            if __dep_name not in self.pkg_dependencies:
+                                self.pkg_dependencies[__dep_name] = set(tuple([]))
+                            _name = name_from_path(name)
+                            if _name not in self.pkg_dependencies:
+                                self.pkg_dependencies[_name] = set(tuple([]))
+                            self.pkg_dependencies[_name].add((_dep_name, dep_version))
```

Comment:
`validate_dependencies` provides a convenient place to extract the dependencies. I'll try and fix the code to avoid changing the `validate_dependencies` function.

Reply:
Done.